tests/test_config.py

```diff
@@ -134,7 +134,7 @@ def test_reading_config_from_other_directory(temp_dir):
     assert config.categories == ["New", "Different", "Gone", "Bad"]
 
 
-def test_unknown_option():
+def test_unknown_option(temp_dir):
```

Comment:
Instead of adding an argument that linters would dislike as unused, you can do this.
```suggestion
@pytest.usefixtures('temp_dir')
def test_unknown_option():
```

Reply:
But no linter seems to mind.